tests/foreman/cli/test_host.py

```diff
@@ -2163,6 +2164,35 @@ def _register_client(
                 result = self.client.run('subscription-manager attach --auto')
 
         if attach_to_default:
+
+            # Remove repository-set from SetUpClass to have no repositories available
+            # Firstly the content view needs to be removed
+            default_cv = ContentView.info({'name': DEFAULT_CV, 'organization-id': self.org['id']})
+            default_lce = LifecycleEnvironment.info(
+                {'name': ENVIRONMENT, 'organization-id': self.org['id']}
+            )
+            ContentView.remove(
+                {
+                    'id': self.content_view['id'],
+                    'lifecycle-environments': f"{ENVIRONMENT},"
+                    + f"{self.env['name']},"
+                    + f"{self.hosts_env['name']}",
```

Comment:
Could you move that to variable that is set before `ContentView.remove`? It's really hard to read.

```python
lcenvs = ",".join([ENVIRONMENT, self.env['name'], self.hosts_env['name'] ])
```

Reply:
I was told to use f string, but I agree it's hard to read it. 
Yes I should use my mind, and should choosethe different approach when the f string is on multiple lines :+1: